consensus.md

```diff
@@ -27,22 +27,22 @@ An `accept` judgement can be changed into a `deny` judgements at a later point i
 Each admission will base its decision on the following rules:
 
  1. Is the block itself correct? (hash, index, transactions, size,...)
- 2. Is the block created by the expected creator (refer to [Creator Election](#creator-election))?
+ 2. Is the block created by the expected creator? (refer to [Creator Election](#creator-election))
  3. Is the block creation time older than every alternative block with the same index?
  
 If all 3 questions can be answered with yes (again based on the current state), the admission will create an `accept` judgement.
 At the same time every alternative branch (every sub-tree with the same index as root) will be updated to a `deny` judgement for every block in the branch.
-If one of the questions is answered with `false` the block hat and will get a `deny` judgement.
+If one of the questions is answered with no the block will get a `deny` judgement.
 
 As you can see every admission has only one active branch at every point in time.
-Since blocks and judgements are broadcasted in the network every admission will base its decision on the same information over time.
+Since blocks and judgements are broadcast in the network every admission will base its decision on the same information over time.

```

Comment:
broadcasted

Reply:
the verb `broadcast` does not conjugate to `broadcasted` but `broadcast`, 
see http://conjugator.reverso.net/conjugation-english-verb-broadcast.html

Reply:
https://dict.leo.org/pdf/flectionTable.pdf?kvz=4dkrADn71HeCbYCs67UP8lumz53RnXH31_iKUb_9hScDG0HfGwQNAs0iazuQS3MykZxvZlpa1HHSa1Dc6nR74kcHyaoRbdcTANyc1_09AVFVbsZcmwVqRAvHXGAGaFZ&lp=ende&lang=de

Reply:
looks like `broadcasted` is also accepted now by the big dictionaries. As `cast` is an irregular verb I'd still vastly prefer to use `broadcast` which is also the more common way of spelling (source: https://english.stackexchange.com/questions/33207/broadcast-or-broadcasted).